internal/dblist/db_list.go

```diff
@@ -99,10 +99,12 @@ func JobList(ctx context.Context, exec riverdriver.Executor, params *JobListPara
 
 	for i, orderBy := range params.OrderBy {
 		orderByBuilder.WriteString(orderBy.Expr)
-		if orderBy.Order == SortOrderAsc {
+		switch orderBy.Order {
+		case SortOrderAsc:
 			orderByBuilder.WriteString(" ASC")
-		} else if orderBy.Order == SortOrderDesc {
+		case SortOrderDesc:
 			orderByBuilder.WriteString(" DESC")
+		case SortOrderUnspecified:
```

Comment:
Maybe this should actually be an error 🤔

Reply:
Ah yeah, I wasn't sure if this was supposed to be possible or not.

I just replaced it with an error instead. Seems to pass the tests so hopefully it's okay ...